tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Pulling the tslint.json from xterm.js would be good https://github.com/xtermjs/xterm.js/blob/master/tslint.json (I can do this after if there are a lot of changes)